apps/consent-ui/src/middleware/apiProxy.ts

```diff
@@ -18,37 +18,33 @@
  */
 
 import { NextRequest, NextResponse } from 'next/server';
-import urlJoin from 'url-join';
-import { AxiosHeaders } from 'axios';
-
-import { consentApiClient } from 'src/services/api';
-import { getRequestData } from 'src/services/api/utils';
+import { auth } from 'src/app/auth';
+import { getAppConfig } from 'src/config';
+import { decryptContent } from 'src/services/api/utils';
 
 /**
- * Route handler for creating proxy requests from /api routes to public consent-api endpoints
- * @param req NextRequest obj
- * @param routePaths path params interpolated from app/api/[...proxy] directory structure
- * @returns response obj from consent-api as NextResponse
+ *
+ * @param request
+ * @returns
  */
-const handler = async (
-	req: NextRequest,
-	routePaths: { params: { proxy: string[] } }, // "proxy" key matches the [...proxy] dynamic path
-): Promise<NextResponse<any>> => {
-	const path = urlJoin(routePaths.params.proxy);
-	const requestData = await getRequestData(req);
-	// TODO: are there any other headers needed here?
-	const headers = new AxiosHeaders();
-	headers.set('Content-Type', 'application/json');
-	// Don't forward cookies to the API:
-	headers.set('Set-Cookie', '');
-	const res = await consentApiClient(path, {
-		method: req.method,
-		headers,
-		...(requestData && { data: requestData }),
-	});
+const apiProxy = async (request: NextRequest) => {
+	// Build url from incoming request
+	const redirectPath = request.nextUrl.pathname.replace(/^\/api/, '');
```

Comment:
I think this regex will need to be updated from `/api` that we've changed `PROXY_API_PATH` to `/consent-api`:
https://github.com/OHCRN/platform/blob/2f84a20b6308e261d2d24eba93d501a055136d14/apps/consent-ui/src/constants/internalPaths.ts#L25

Since the middleware/proxy is now being called based on `/consent-api` as the path:
https://github.com/OHCRN/platform/blob/2f84a20b6308e261d2d24eba93d501a055136d14/apps/consent-ui/src/middleware.ts#L55

Probably a simple fix like this (note you'll need to import `PROXY_API_PATH` as well):
```suggestion
	const proxyPathRegex = new RegExp(`^${PROXY_API_PATH}`);
	const redirectPath = request.nextUrl.pathname.replace(proxyPathRegex, '');
```